scripts/run-in-pipeline.sh

```diff
@@ -0,0 +1,194 @@
+#!/bin/sh
+##
+#  This script gets packaged with a Redhat python UBI (Universal Base Image) and serves as the entrypoint of the container.
+#  The IBM Detect Secrets CLI tool is also packaged in the container; these scripts will run:
+#      1) Detect Secrets scan --update
+#      2) followed by:  Detect Secrets audit -report with specified fail-on-xx fails
+#  to generate a report and optionally return a failing return code on selected conditions base on the specified fail-on-xx flags.
+#
+#  Note:  We include a SKIP_SCAN flag if one desired to skip the scan --update part (step 1 above) and just to the audit -report with existing baseline
+#         This option might be chosen in the case that they alrady have an updated baseline from you the Detect Secrets CLI dev tool.
+#
+#  This script take is paramenter via Environment Variables:
+#    BASELINE :  This is the filename to use to store/read the scan baseline output/input.
+#      * Default:  .secrests.baseline
+#
+#    SKIP_SCAN : (True/False) Allows for the skipping of the Scan --update part that updates the baseline file.
+#      * Default:  False
+#
+#    JSON : (True/False) Whether to output result in JSON format vs the Table format
+#      * Default:  False
+#        Note:  Setting True is mutually exclusive with setting OMIT_INSTRUCTIONS=True.
+#
+#    OMIT_INSTRUCTIONS : (True/False) Whether to output resolution instruction for failed conditions.
+#      * Default:  False
+#        Note:  Setting True is mutually exclusive with setting JSON=True.
+#
+#    FAIL_ON_LIVE : (True/False) Sets the condition to fail audit if there are verified live secrets found
+#      * Default:  True
+#
+#    FAIL_ON_UNAUDITED : (True/False) Sets the condition to fail audit if there are unaudited potential secrets found
+#      * Default:  True
+#
+#    FAIL_ON_AUDITED_REAL : (True/False) Sets the condition to fail audit if there are audited and set to real secrets found
+#      * Default:  True
+#
+# Example manual docker run within pacakaged container (git-defenders/detect-secrets-rhubi):
+#    docker run  --env BASELINE=.secrets.baseline  --env FAIL_ON_LIVE=False  -it -a stdout --rm -v $(pwd):/code git-defenders/detect-secrets-rhubi
+##
+
```

Comment:
Thanks for taking the time to document what this script is doing! This is very helpful.